drivers/input/Kconfig

```diff
@@ -14,7 +14,7 @@ menuconfig INPUT
 if INPUT
 
 config INPUT_MOUSE
-	bool "Enable mouse support"
```

Comment:
Why the text is removed?

Reply:
INPUT_MOUSE is only a flag to indicate we have mouse support, it is activated by other device drivers that implement some kind of "mouse". Note there is not a mouse.c file or equivalent to be compiled when INPUT_MOUSE is enabled

Reply:
If it is a "hidden" bool shouldn't the help go away also?

Reply:
Even it's a hidden option, the mouse driver dev can get help from it when he add new driver option to Kconfig file.